CHANGES.rst

```diff
@@ -116,6 +116,8 @@ Bug Fixes
 
 - ``astropy.io.ascii``
 
+  - Strip leading/trailing white-space from latex lines to avoid issues when matching `\begin{tabular}` statements.  This is done by introducing a new `LatexInputter` class to override the `BaseInputter`.
```

Comment:
Use double backticks otherwise sphinx tries to links these but fails and thus raises the warnings.